models/donation.model.js

```diff
@@ -3,7 +3,13 @@ const mongoose = require('mongoose');
 const DonationSchema = new mongooose.Schema({
   user: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
   ngo: { type: mongoose.Schema.Types.ObjectId, ref: "NGOs" },
-  address: { type: String, require: true },
+  location: {
```

Comment:
@akshitbansal108 @PragsJ  make changes in controllers .

Reply:
@kiranbhanushali okay, will do.